### PR TITLE
Version update to substrate 8a9a8f

### PR DESCRIPTION
**Pull request by haerdib**
Updates sgx-runtime to support substrate commit ~~`e5437efefa82bd8eb567f1245f0a7443ac4e4fe7`~~

EDIT CL: bump substrate to 8a9a8f

### CODE REVIEW

**Review by clangenb — COMMENTED** [review]
looks good to me

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -88,7 +88,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node-template"),
 	impl_name: create_runtime_str!("node-template"),
 	authoring_version: 1,
-	spec_version: 1,
+	spec_version: 100,
```
Comment (brenzi) [inline runtime/src/lib.rs:91]: what is this version bump about?

Comment (clangenb) [reply runtime/src/lib.rs:91]: Good question, I thought this was something you and @haerdib coordinated.

Comment (haerdib) [reply runtime/src/lib.rs:91]: An adaption to the changes substrate has done on their node-template: 
https://github.com/paritytech/substrate/blob/master/bin/node-template/runtime/src/lib.rs#L100

resp, the PR: 
https://github.com/paritytech/substrate/pull/8473

Comment (haerdib) [reply runtime/src/lib.rs:91]: Should we change back to 1?

Comment (clangenb) [reply runtime/src/lib.rs:91]: Well in our case this is currently only informative, but I still think 1 is better. 100 seems a bit random.

Comment (brenzi) [reply runtime/src/lib.rs:91]: If the comment
> 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
>	//   the compatible custom types.
is the only reason to bump it I'm not sure it serves us. What "custom types" are these? Just template stuff?

I'm in favor of `1`

**Pull request merged**